cvat-canvas/README.md

```diff
@@ -0,0 +1,93 @@
+## Module
+
+- Written on typescript
+- Contains the class ```Canvas```
+
+## Creation
+Canvas is created by using constructor:
+
+```js
+    const canvas = new Canvas();
+```
+
+- Canvas has transparent background
+
+Canvas itself handles:
```

Comment:
Probably UI should be able to provide alternative shortcuts. It should be a reasonable default.

Reply:
@nmanovic 
It is easy to setup alternative shortcut to "fit", for example. But I don't have any ideas about how it should work with image moving for example. 
In other cases only mouse can be used (and here we just do not have alternatives).